types/emartech__cls-adapter/tsconfig.json

```
@@ -0,0 +1,29 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "paths": {
+            "@emartech/cls-adapter": [
+                "emartech__cls-adapter"
+            ]
+        },
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

```suggestion
        "forceConsistentCasingInFileNames": true
```
IMO This is not required here and not adviced for DT definition itself, unless there is a specific reason to include this.
See:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44595#discussion_r425400276